pghoard/basebackup.py

```diff
@@ -780,11 +780,17 @@ def run_local_tar_basebackup(self, delta=False):
                 db_conn.commit()
                 backup_stopped = True
 
+                backup_time = time.monotonic() - start_time
+                self.metrics.gauge(
+                    "pghoard.backup_time_{}".format(self.site_config["basebackup_mode"]),
+                    backup_time,
+                    tags={"site": self.site}
```

Comment:
I don't think site is very interesting value as a tag. In our context it's random and not good for metrics cardinality

Reply:
Yep, I agree, will remote it, I've just seen it for other metrics.